src/ecrc-frontend/src/components/page/applicationForm/ApplicationForm.js

```diff
@@ -308,6 +326,7 @@ export default function ApplicationForm({
 
   return (
     <main>
+      {toHome ? <Redirect to="/" /> : null}
```

Comment:
makes sense that we are redirecting if toHome, but what is the `null` part?

Reply:
is it required

Reply:
The null is part of the ternary statement so while toHome is false it renders nothing. Im not sure if you can put a traditional if statement in there, but I think there is a way to move it outside if you prefer?
Would be something like:
if (toHome) {
  return <Redirect...>
}

as a function before the regular return statement

Reply:
lets redirect within the functions themselves to avoid this 😄